README.md

```diff
@@ -43,11 +43,26 @@ The `italia-app` application will authenticate to the backend in two steps:
   1. an initial user initiated SPID authentication process (SAML2 based)
      that identifies the user and, on success, triggers the creation of a new
      authentication session (associated to a session token)
-  2. subsequent API requests authenticated via a bearer session token
+  2. subsequent requests to the backend will be authenticated via a bearer session token
+  
+See the sequence diagram [here](https://www.planttext.com/?text=PO_H2e9044NVzolc1w9teJ9KQa5qqUrPH8TOqDrbTepy-xOY47eSEFVSEsdWddYP9o7EtiFk28hSSqtWKcZOJ-vkx0aj6rvbVZ2Zxxv7PqOXFkGx8dELm5Lh1OIZ8HmOs44aQLNEM1OY4XktgjiexqD0OkDivcnOPgGMwM46t8X_Mz0YzOomn0G4326cDbXS_jVl8IjbKUckANBP59cMTTNTcbA85EqO7tu1)
```

Comment:
I suggest to download it as SVG (it will embed the plantuml source automatically) and save it as an asset in the repository

Reply:
Good to know, I'll embed the svg

Reply:
@gunzip what is the process of editing/modifying the diagram once you download the SVG?

Reply:
diagram source is in the svg file so the process is: copy from svg to a new diagram on plantuml and regenerate the svg file